lib/ResourceSet.h

```diff
@@ -214,7 +214,25 @@ class ResourceSet
 
 	template <typename Handler> void serialize(Handler &h)
 	{
-		h & container;
+		if (h.version >= Handler::Version::CONFIGURABLE_RESOURCES)
+			h & container;
+		else
+		{
+			if (h.saving)
```

Comment:
Ok to merge, but there is no need to implement saving of old save format. We only need to load old save